browser/services/platform.js

```diff
@@ -122,6 +122,14 @@ class Platform {
     });
   }
 
+  static removedirectory(dir) {
+    return pify(child_process.exec)('rm -rf ' + dir).then((stdout) => {
```

Comment:
This method is not necessarry, you can use fs-extra#remove or rimraf to the same effect. Plus rm is not safe to use under windows.